File: olo-promotions-sdk-go/pkg/promotions/models/responses/promotions/validate_promotions_response.go
package promotions

import "olo-promotions-sdk-go/pkg/promotions/models/responses"

// ValidatePromotionsResponse is returned for a successful call to the Validate Promotions endpoint.
type ValidatePromotionsResponse struct {
	// Transaction contains the transaction details along with validated promotions.
	Transaction responses.TransactionWithPromotions `json:"transaction"`
}

File: olo-promotions-sdk-csharp/Olo.Promotions.SDK/Responses/ValidatePromotionsResponse.cs
using Olo.Promotions.SDK.Responses.Models;

namespace Olo.Promotions.SDK.Responses
{
    /// <summary>
    /// Returned for a successful call to the Validate Promotions endpoint.
    /// </summary>
    public class ValidatePromotionsResponse
    {
        public TransactionWithPromotions Transaction { get; set; }
    }
}